src/liveEditor/utils/__test__/getExpectedFieldData.test.ts

```diff
@@ -0,0 +1,24 @@
+import { LiveEditorPostMessageEvents } from "../types/postMessage.types";
+import { getExpectedFieldData } from "../getExpectedFieldData";
+
+jest.mock("../liveEditorPostMessage", () => {
+    return {
+        __esModule: true,
+        default: {
+            send: jest.fn().mockImplementation((eventName: string) => {
+                if (eventName === LiveEditorPostMessageEvents.GET_FIELD_DATA)
+                    return Promise.resolve({
+                        fieldData: "I am the correct long expected data",
+                    });
+                return Promise.resolve();
+            }),
+        },
+    };
+});
```

Comment:
Instead of mocking this module, take a look at how we handle this module in other test files.